tools/sync_test_server/ros-testing-server.js

```diff
@@ -64,10 +65,16 @@ function stopRealmObjectServer() {
     if (syncServerChildProcess) {
         syncServerChildProcess.kill();
         syncServerChildProcess = null;
+        exec('rm -r ' + 'realm-object-server', function (err, stdout, stderr) {
```

Comment:
really? It is not creating this dir in `--root path`? That sounds incorrect .....